### PR TITLE
SUP-180: local footer

### PR DESCRIPTION
**Pull request by jenbreese**
#  READY FOR REVIEW
- Feedback requested

# Summary
- Styling
-- Changed the bg color
-- Changed the link colors
- Responsive
-- Reordered content
- Removed the address
- Moved the social icons
- Removed the Lockup options
- Added the logo svg for the footer.
- Removed the red SU global footer.
- Removed the Lockup stories file since they won't change it.
- Known things to change.
-- Spacing below the `h2` to match design
-- Spacing on the columns between the lines and the center column of links. 
-- Change the m1 font sizing to the type-0 to type-10. This is more up to date way to have the responsive font sizing. 

# Review By (Date)
- When time

# Criticality
- Normal

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
5. Rebuild Cache and import config `drush cr ; drush ci`
6. Navigate to the footer config page.  
7. Add the links as listed in the [ticket](https://stanfordits.atlassian.net/browse/SUP-180) to the 4 different content blocks. There is a .txt file in the Jira ticket with all the links from the design.
8. Remove the address.
9. Change the social icons to include an Instagram. 
10. Verify the link lists order changes in the different responsive views. 

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- SUP-180](https://stanfordits.atlassian.net/browse/SUP-180)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/styles/fonts.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import {Source_Sans_3} from "next/font/google";
+import {Montserrat} from "next/font/google";
 
-export const sourceSans3 = Source_Sans_3({
+export const sourceSans3 = Montserrat({
```
Comment (pookmish) [inline src/styles/fonts.tsx]: If you change the font, you should change the variable name here and in the `layout.tsx` where it is used.
I'll fix it though.

File: src/components/images/logo-lg.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import {HtmlHTMLAttributes} from "react";
+
+type Props = HtmlHTMLAttributes<SVGSVGElement> & {
+  height?: number
+  width?: number
+}
+
+const PressFooterLogoLg = ({height = 89, width = 579, ...props}: Props) => {
```
Comment (pookmish) [inline src/components/images/logo-lg.tsx:8]: You should add `{...props}` to the `svg` attributes.

File: src/components/images/logo-xs.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import {HtmlHTMLAttributes} from "react";
+
+type Props = HtmlHTMLAttributes<SVGSVGElement> & {
+  height?: number
+  width?: number
+}
+
+const PressFooterLogoXs = ({height = 91, width = 426, ...props}: Props) => {
```
Comment (pookmish) [inline src/components/images/logo-xs.tsx:8]: You should add `{...props}` to the `svg` attributes.

**Review by pookmish — APPROVED** [review]
I'll make the necessary changes.

File: src/styles/fonts.tsx (hunk 1 above)
Comment (jenbreese) [reply src/styles/fonts.tsx]: Oh right, good point.

File: src/components/images/logo-lg.tsx (hunk 2 above)
Comment (jenbreese) [reply src/components/images/logo-lg.tsx:8]: Ok.  Thanks for the feedback!

**Pull request merged**